frontend/package.json

```diff
@@ -35,7 +35,9 @@
     "webpack-cli": "^5.1.4",
     "webpack-dev-server": "^4.15.1"
   },
-  "dependencies": {},
+  "dependencies": {
```

Comment:
@punchagan Just noticed this.. Should I install as a dev dependency here?

Reply:
For a project that's being completely built as a bundle, I wasn't sure what the difference between dev dependencies and actual dependencies would be. If we really care about it, we could separate out things at some point.

Reply:
ahh cool even im not well versed at that. Will keep like this for now i guess